src/background/services/seedless/SeedlessBtcSigner.ts

```diff
@@ -5,6 +5,7 @@ import {
 } from '@avalabs/core-wallets-sdk';
 import { SignerSession } from '@cubist-labs/cubesigner-sdk';
 import { Network, Psbt, SignerAsync, payments } from 'bitcoinjs-lib';
+import { TransactionInput, TransactionOutput } from 'bip174/src/lib/interfaces';
```

Comment:
Hmm, `bip174` is not a direct dependency of the extension; it's only there because of other dependencies, so I'm not sure if we can rely on it to be there & `import` it like that.

Also, I'm pretty sure updating `@avalabs/avalanchejs` to the same version that's used by vm modules (`4.2.0-alpha.3`) will fix those typings issues. 🤔